lab_3_generate_by_ngrams/main.py

```diff
@@ -129,6 +200,13 @@ def fill_from_ngrams(self, content: dict) -> None:
         Args:
             content (dict): ngrams from external JSON
         """
+        if not(isinstance(content, dict) and content):
+            return None
+
+        n_grams = ''.join([''.join((filter(lambda x: x.isalpha, key))) for key in content['freq']])
```

Comment:
I believe spaces are to be replaces with eos token, and then we should filter by letters